### PR TITLE
Skeleton models for Investments

### PR DESCRIPTION
**Pull request by r4vi**
Includes models for investments and the corresponding models required
for the importer

### CODE REVIEW

File: fdi/models/importer.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    class Meta:
+        unique_together = ('import_id', 'row_index',)
+
+
+class InvestmentLegacyLoad(models.Model):
+    """
+    From Tord's historical spreadsheets
+    """
+    filename = models.CharField(max_length=255)
+    row_index = models.PositiveSmallIntegerField(blank=False, null=False)
```
Comment (r4vi) [inline fdi/models/importer.py]: not sure if row_index is required

File: fdi/models/live.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+
+class Investments(models.Model):
+    """
+    The model to query all MI data from
```
Comment (r4vi) [inline fdi/models/live.py]: we can flesh this out later once designs are done

File: fdi/models/importer.py (hunk 1 above)
Comment (sekharpanja) [reply fdi/models/importer.py]: It doesn't hurt, lets keep it.

File: fdi/models/live.py (hunk 2 above)
Comment (sekharpanja) [reply fdi/models/live.py]: Agreed

**Pull request merged**